<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>image</class>
 <widget class="QDialog" name="image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>551</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>500</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
  </widget>
  <widget class="QLabel" name="sliderValueLBL">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="chemin_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dialValueLBL">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>280</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="left_LBL">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>470</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="rigth_LBL">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>470</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="ctrl_LBL">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>510</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>CTRL Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>691</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>630</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: &quot;red&quot;;

padding: 5px;
border-radius: 5px;

font-weight: bold;

</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <action name="actionZoomInOut">
   <property name="text">
    <string>ZoomInOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom Image</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Ruot</string>
   </property>
  </action>
  <zorder>pic</zorder>
  <zorder>graphicsView</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>dial</zorder>
  <zorder>sliderValueLBL</zorder>
  <zorder>chemin_2</zorder>
  <zorder>dialValueLBL</zorder>
  <zorder>left_LBL</zorder>
  <zorder>rigth_LBL</zorder>
  <zorder>ctrl_LBL</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>image</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1128</x>
     <y>646</y>
    </hint>
    <hint type="destinationlabel">
     <x>1218</x>
     <y>635</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
